src/ILVerification/src/ILVerifyTypeSystemContext.cs

```diff
@@ -32,7 +32,7 @@ public override ModuleDesc ResolveAssembly(AssemblyName name, bool throwIfNotFou
         {
             // Note: we use simple names instead of full names to resolve, because we can't get a full name from an assembly without reading it
             string simpleName = name.Name;
-            return ResolveAssemblyOrNetmodule(simpleName, simpleName, throwIfNotFound);
+            return ResolveAssemblyOrNetmodule(simpleName, simpleName, null, throwIfNotFound);
         }
 
         internal override ModuleDesc ResolveModule(ModuleDesc referencingModule, string fileName, bool throwIfNotFound = true)
```

Comment:
I missed this one in my change - I think we should change the `referencingModule` from `ModuleDesc` to `IAssemblyDesc`. The one place that calls this in `EcmaModule` should actually get it's `Assembly` property and pass that (the line is currently `return Context.ResolveModule(this, fileName);`, but should be `return Context.ResolveModule(this.Assembly, fileName);`